torchtune/modules/transformer.py

```diff
@@ -429,20 +429,13 @@ def setup_caches(
                 decoder_max_seq_len=self.decoder_max_cache_seq_len,
             )
 
-    def encoder_caches_are_enabled(self) -> bool:
-        """Checks if there are any :class:`~torchtune.modules.TransformerCrossAttentionLayer`,
-        or :class:`~torchtune.modules.fusion.FusionLayer` layers which have cache enabled.
-        """
-        return self.encoder_max_cache_seq_len is not None
-
-    def decoder_caches_are_enabled(self) -> bool:
-        """Check if there are any :class:`~torchtune.modules.TransformerCrossAttentionLayer`
-        layers which have cache enabled."""
-        return self.decoder_max_cache_seq_len is not None
+    def caches_are_enabled(self) -> bool:
+        """Check if the key value caches are setup."""
```

Comment:
nit: fine as it is, but maybe add context about how/when this is used. Something like: "useful during inference to xyz". Feel free to ignore